test/services/EtmpRegimeServiceTest.scala

```diff
@@ -462,4 +495,50 @@ class EtmpRegimeServiceTest extends BaseSpec {
       }
     }
   }
+
+  "fetchETMPStatus" should {
+    "provide the status of the registration if it already exists" when {
+      "a rejected case" in {
+        val resp = Json.obj(
+          "processingDate" -> "20/01/2010",
+          "formBundleStatus" -> Rejected.name,
```

Comment:
Can you use a string here "Rejected" because it gets set to lower case in the model so would be a more accurate test ensuring the case sensitivity isn't an issue.